packages/slice-machine/server/src/api/index.ts

```diff
@@ -174,16 +174,16 @@ router.get(
 router.put(
   "/slices/rename",
   // eslint-disable-next-line @typescript-eslint/no-misused-promises
-  WithEnv(async function (
+  WithEnv(function (
```

Comment:
Curious about this change. What triggered it?

Reply:
A function call inside `generateLibraryIndex` was being awaited for no reason. And had an eslint disable `await-thenable`